site/layouts/partials/footer.html

```diff
@@ -1,56 +1,113 @@
+                     aria-describedby="headingGuides">Starter template</a></li>
+              <li><a class="nav-link" href="/docs/{{ .Site.Params.docs_version }}/getting-started/webpack/"
+                     aria-describedby="headingGuides">Webpack</a></li>
+              <li><a class="nav-link" href="/docs/{{ .Site.Params.docs_version }}/getting-started/parcel/"
+                     aria-describedby="headingGuides">Parcel</a></li>
+            </ul>
+          </div>
+        </div>
+        <div class="footer-column col-md-2">
+          <h3 class="accordion-header footer-heading" id="headingProjects">
+            <button class="accordion-button collapsed container-xxl px-1 d-md-none" type="button"
+                    data-bs-toggle="collapse" data-bs-target="#collapseProjects" aria-expanded="true"
+                    aria-controls="collapseProjects">
+              Projects
+            </button>
+            <span class="d-none d-md-flex">Projects</span>
+          </h3>
+          <div id="collapseProjects" class="container-xxl accordion-collapse collapse" data-bs-parent="#sitemapAccordion">
+            <ul class="navbar-nav">
+              <li><a class="nav-link" href="{{ .Site.Params.repo }}" aria-describedby="headingProjects">Boosted 5</a>
+              </li>
+              <li><a class="nav-link" href="{{ .Site.Params.repo }}/tree/v4-dev" aria-describedby="headingProjects">Boosted
+                4</a></li>
+              <li><a class="nav-link" href="{{ .Site.Params.bootstrap }}"
+                     aria-describedby="headingProjects">Bootstrap</a></li>
+              <li><a class="nav-link" href="https://boosted.netlify.app" aria-describedby="headingGuides">Pre-release on
+                Netlify</a></li>
+            </ul>
+          </div>
+        </div>
+        <div class="footer-column col-md-2">
+          <h3 class="accordion-header footer-heading" id="headingFour2">
+            <button class="accordion-button collapsed container-xxl px-1 d-md-none" type="button"
+                    data-bs-toggle="collapse" data-bs-target="#collapseCommunity" aria-expanded="true"
+                    aria-controls="collapseCommunity">
+              Community
+            </button>
+            <span class="d-none d-md-flex">Community</span>
+          </h3>
+          <div id="collapseCommunity" class="container-xxl accordion-collapse collapse" data-bs-parent="#sitemapAccordion">
+            <ul class="navbar-nav">
+              <li><a class="nav-link" href="{{ .Site.Params.repo }}/issues" aria-describedby="headingFour2">Issues</a>
+              </li>
+              <li><a class="nav-link" href="{{ .Site.Params.repo }}/discussions" aria-describedby="headingFour2">Discussions</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <div class="footer-column col-md-3 mb-3 pt-3 pt-md-4 px-1 px-md-0 fw-bold small">
```

Comment:
These paragraphs might be under the back-to-top button in some cases, we could maybe try to handle it in a different way ? + in small viewports, missing accordion border between these paragraphs and the last accordion item. Maybe we should improve the the footer CSS to handle this kind of cases ? (not sure about that)

Reply:
Like we said, this problem with back to top must be processed in another issue

Reply:
Border has been added in small viewports